data/download.yaml

```diff
@@ -0,0 +1,42 @@
+
+binaries:
+  platforms:
+  - darwin
+  - linux
+  - windows
+  platformInfo:
+    darwin:
+      name: macOS
+      icon: fa-apple
+    linux:
+      name: Linux
+      icon: fa-linux
+    windows:
+      name: Windows
+      icon: fa-windows
+  fullReleaseVersions:
+  - "1.4.1"
+  - "1.5.0"
+  - "1.6.0"
+  partialReleaseVersions:
+  - "1.4.0"
+  - "1.3.0"
+  - "1.2.0"
+  - "1.1.0"
+  - "1.0.0"
+  - "0.9.0"
+  - "0.8.0"
+  - "0.7.0"
+  - "0.6.0"
+  - "0.5.2"
+  - "0.5.1"
+  - "0.5.0"
+docker:
+- name: jaeger-agent
+  description: The Jaeger agent
+- name: all-in-one
+  description: The "all-in-one" hot rod example used in the [getting started](/docs/latest/getting-started) guide
+- name: jaeger-collector
+  description: The Jaeger collector
+- name: jaeger-query
+  description: The Jaeger query image
```

Comment:
should this table be titled "for example"? Because it misses many other Docker images like HotROD, Spark job, DB schema utils, etc. If we do want to list everything (which wouldn't be too bad since our DockerHub doesn't have much documentation), then the question is whether the explanation should be elsewhere. The archives with binaries, for example, do not include many of those additional components.

Reply:
That's up to you. Perhaps we can merge this and you can follow up by adding the images and descriptions that you'd want to see here?